### PR TITLE
Use rewired's injectBabelPlugin

### PR DESCRIPTION
**Pull request by mdebbar**

### CODE REVIEW

File: packages/react-app-rewire-emotion/index.js [hunk 1]
```diff
@@ -1,22 +1,7 @@
...
-    return !babelLoader;
-  });
-
-  if (babelLoader && babelLoader.options) {
-    babelLoader.options.plugins = (babelLoader.options.plugins || []).concat([['emotion/babel', emotionBabelOptions]]);
-  } else if (babelLoader && babelLoader.query) {
-    babelLoader.query.plugins = (babelLoader.query.plugins || []).concat([['emotion/babel', emotionBabelOptions]]);
-  }
+function rewireEmotion(config, env) {
+  config = injectBabelPlugin('emotion/babel', config);
```
Comment (osdevisnot) [inline packages/react-app-rewire-emotion/index.js]: Unfortunately this breaks the React Storybook setup for me. React storybook uses older version of webpack and babel loader there goes in babelLoader.query.plugins and rewires's injectBabelPlugin does not add it to babelLoader.

Comment (mdebbar) [reply packages/react-app-rewire-emotion/index.js]: Hmm I see. Ok, I'll make a PR to change `injectBabelPlugin` in rewired.

Comment (osdevisnot) [reply packages/react-app-rewire-emotion/index.js]: :100:

Comment (mdebbar) [reply packages/react-app-rewire-emotion/index.js]: For reference: https://github.com/timarney/react-app-rewired/pull/91

**Comment by mdebbar** [comment]
Hey @osdevisnot, mind taking a look again? My [PR to react-app-rewired](https://github.com/timarney/react-app-rewired/pull/91) was merged! I also made the same changes for the lodash and styled packages.

**Comment by osdevisnot** [comment]
@mdebbar Merged and released. Apologies for delays, I did not realize the other PR is already merged.

**Pull request merged**